Vagrantfile

```diff
@@ -87,55 +125,58 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     box
   end
   config.vm.box_url = box_url
-  config.vm.network :private_network, :ip => "10.10.10.30"
+  config.vm.network :private_network, ip: SETTINGS['ip']
 
-  if ALAVETELI_USE_NFS
-    config.vm.synced_folder ".", "/home/vagrant/alaveteli", :nfs => true
+  if SETTINGS['use_nfs']
```

Comment:
Can't override this on the command line if it's on in the .vagrant.yml file (pretty sure this is how it was working before)